src/main/java/com/fundation/webservice/model/KeyFrameOfVideo.java

```diff
@@ -18,17 +19,12 @@
  * @author Josue Rodriguez
  * @version 1.0
  */
-public class KeyFrameOfVideo {
-    CriteriaKeyFrameVideo criteria;
-
-    public KeyFrameOfVideo(CriteriaKeyFrameVideo criteria) {
-        this.criteria = criteria;
-    }
-
+public class KeyFrameOfVideo implements IConvert{
     /**
      * Converts method get the keyframe depending of the number of frame.
      */
-    public void convert() {
+    public void convert(CriteriaConvert criteriaConvert) {
+        CriteriaKeyFrameVideo criteria = (CriteriaKeyFrameVideo)criteriaConvert;
         try {
             String cmd = Directories.TOOLS_DIR.getDir() + Directories.FFMPEG_DIR.getDir() + "ffmpeg -y -i "
                 + criteria.getSrcPath() + " -vf fps=1/" + criteria.getTime() + " " + criteria.getDestPath()
```

Comment:
What is fps=1 its looks magical numbers?

Reply:
it is not a magical number is a command defect value.

Reply:
That part is a string that is being feed onto the command line to ffmpeg